src/main/java/org/jabref/logic/importer/util/MetaDataParser.java

```diff
@@ -97,6 +100,77 @@ public MetaData parse(Map<String, String> data, Character keywordSeparator) thro
         return parse(new MetaData(), data, keywordSeparator);
     }
 
+    public MetaData parse(JsonObject data, Character keywordSeparator) throws ParseException {
```

Comment:
Add JavaDoc - I don't get this.

I think, this should not be in this PR for now - We only focus on the JSON - and then, we can work on the parsing of the JSON.